eva-accession-release/src/main/resources/copy_release_to_ftp.sh

```diff
@@ -0,0 +1,226 @@
+#!/bin/bash
+# Copyright 2019 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: copy_release_to_ftp.sh <input_folder> <output_folder> <intermediate_folder> <unmapped_variants_folder>
```

Comment:
Please add "set -euf -o pipefail" to the top of the script since it guards against so much brain-damaged behavior that bash brings. See https://sipb.mit.edu/doc/safe-shell/